interfaces/queryObject.tsx

```diff
@@ -0,0 +1,5 @@
+export interface queryObject {
+  orderBy?: undefined | string;
+  q?: undefined | string;
+  rl?: undefined | string | string[];
```

Comment:
Would be good to add a comment of what q and rl means.

Reply:
Add comment for each key-value pair